### PR TITLE
Fix spelling & formatting errors

### PR DESCRIPTION
**Pull request by hendursaga**
Just a # of minor spelling errors as well as some reformatting (while connected to a running Nyxt instance, of course).

### CODE REVIEW

File: source/mode/download.lisp [hunk 1]
```diff
@@ -81,7 +81,7 @@ the URL. It will search the URLs of all the existing downloads, if it
 finds it, it will invoke its cancel-function."
   (alex:when-let ((download (find url (downloads *browser*) :key #'url :test #'equal)))
     (funcall (cancel-function download))
-    (echo "Download cancelled: ~a." url)))
+    (echo "Download canceled: ~a." url)))
```
Comment (aadcg) [inline source/mode/download.lisp:85]: Both versions are acceptable. Let's perhaps leave it as it was?

File: source/mode/hint.lisp [hunk 2]
```diff
@@ -119,9 +119,9 @@ For instance, to include images:
   (let* ((exponents (nreverse (loop for pow below char-length
                                     collect (expt (length alphabet) pow)))))
     (coerce (loop for exp in exponents
-                  for quotinent = (floor (/ code exp))
-                  collect (aref alphabet quotinent)
-                  do (decf code (* quotinent exp)))
+                  for quotient = (floor (/ code exp))
+                  collect (aref alphabet quotient)
+                  do (decf code (* quotient exp)))
             'string)))
```
Comment (aadcg) [inline source/mode/hint.lisp:125]: I can't believe I missed this typo, given how many times I actually looked at this! 

Thanks.

File: source/mode/os-package-manager.lisp [hunk 3]
```diff
@@ -31,7 +31,7 @@
      (current-buffer)
      (ps:ps (ps:chain document
                       (write (ps:lisp (spinneret:with-html-string
-                                        (:p "Operation cancelled.")))))))))
+                                        (:p "Operation canceled.")))))))))
```
Comment (aadcg) [inline source/mode/os-package-manager.lisp:35]: Same as above.

File: source/mode/repeat.lisp [hunk 4]
```diff
@@ -90,7 +90,7 @@ It takes a `repeat-mode' instance as argument.")))
 
 (defun skip-repeat-dispatch (keyspec)
   (declare (ignore keyspec))
-  (echo "Cancelled repeat-key.")
+  (echo "Canceled repeat-key.")
   (setf (command-dispatcher (current-window)) #'dispatch-command
```
Comment (aadcg) [inline source/mode/repeat.lisp:94]: Same.

**Comment by aadcg** [comment]
Thank you @hendursaga for all of these fixes. I've left some minor review comments.

**Comment by hendursaga** [comment]
As detailed [here](https://www.merriam-webster.com/words-at-play/canceled-or-cancelled), the single-L variant is primarily American (like me). It doesn't particularly matter to me either way, but perhaps say [in the programming conventions](https://github.com/atlas-engineer/nyxt/blob/master/documents/README.org#programming-conventions) we could clarify whether either American or British English is preferred, or if neither are??

**Comment by aadcg** [comment]
> As detailed [here](https://www.merriam-webster.com/words-at-play/canceled-or-cancelled), the single-L variant is primarily American (like me). It doesn't particularly matter to me either way, but perhaps say [in the programming conventions](https://github.com/atlas-engineer/nyxt/blob/master/documents/README.org#programming-conventions) we could clarify whether either American or British English is preferred, or if neither are??

I've thought about that as well. I'd stick with american spelling. I'm sure @jmercouris won't oppose.

**Comment by Ambrevar** [comment]
Fine with me.
American spelling is shorter, that's the only argument I have.

**Comment by aadcg** [comment]
@hendursaga would you like to add an entry to our style guide regarding american spelling so that we merge this PR?

**Comment by aadcg** [comment]
Pushed via e538cc048. Thanks @hendursaga.

**Pull request closed without merging**